podman/domain/networks.py

```diff
@@ -1,60 +1,109 @@
-"""Model and Manager for Network resources."""
-from typing import List
+"""Model and Manager for Network resources.
 
-from podman.api.client import APIClient
-from podman.domain.manager import Manager, PodmanResource
+Note:
+    By default, most methods in this module uses the Podman compatible API rather than the
+        libpod API as the results are so different.  To use the libpod API add the keyword argument
+        compatible=False to any method call.
```

Comment:
It's probably "is what it is", but any chance of renaming "compatible" or changing the argument "compatible=podman" or "compatible=libpod"?  
Not necessary for this review, just for further digestion.

Reply:
@TomSweeneyRedHat I'm open to a better name, I went with that since it is what we used on the server.  I'm looking for a boolean and a short-ish name.